usr/share/rear/layout/save/GNU/Linux/210_raid_layout.sh

```diff
@@ -93,7 +130,7 @@ if [ -e /proc/mdstat ] &&  grep -q blocks /proc/mdstat ; then
                 name=""
             fi
 
-            echo "raid ${device}${metadata}${level}${ndevices}${uuid}${name}${sparedevices}${layout}${chunksize}${devices}"
+            echo "raid ${device}${metadata}${level}${ndevices}${uuid}${name}${sparedevices}${layout}${chunksize}${devices}${size}"
```

Comment:
Because `$size` is added
perhaps the "Disk layout file syntax" for "Software RAID" in
https://github.com/rear/rear/blob/master/doc/user-guide/06-layout-configuration.adoc
should also be adapted?